source/Calamari.Shared/Integration/Scripting/PackagedScriptRunner.cs

```diff
@@ -45,25 +45,38 @@ protected void RunScripts(RunningDeployment deployment)
             }
         }
 
-        protected void DeleteScripts(RunningDeployment deployment)
+        protected void DeletePreferredScript(RunningDeployment deployment)
         {
-            var scripts = FindScripts(deployment);
+            var script = FindPreferredScript(deployment);
 
-            foreach (var script in scripts)
```

Comment:
We should still delete all the scripts. It would feel strange if we left behind Octopus artifacts in an application folder.